content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -722,6 +722,92 @@ controller is tracking a Job using Pod finalizers by checking if the Job has the
 annotation `batch.kubernetes.io/job-tracking`. You should **not** manually add
 or remove this annotation from Jobs.
 
+### Specifying your Pod failure policy
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+In some situations you may want to have a better control over handling of Pod
+failures than the default policy based on the Job's `.spec.backoffLimit` gives
+(see: [Pod backoff failure policy](#pod-backoff-failure-policy)).
+
+For example, you may want to terminate a job as soon as one of its Pods fails
+with an exit code indicating a software bug (for example due to an unknown
```

Comment:
I would remove the example within the parenthesis. It's too specific and not very illustrative.

Reply:
ok, I thought it might be useful to note cause it seems to me it might be a common reason which would resonate with many users, but indeed it seems too specific for this paragraph